Makefile

```diff
@@ -140,7 +140,7 @@ GINKGO_NOCOLOR ?= false
 FLAVOR ?= e2e
 
 TEST_NAMESPACE=capx-test-ns
-TEST_CLUSTER_NAME=mycluster
+TEST_CLUSTER_NAME=capx-cl-${USER}
```

Comment:
I would suggest to remove the user parameter and just allow the user to override TEST_CLUSTER_NAME and TEST_NAMESPACE variables

Reply:
could you please elaborate on your concern with usernames? 
i can use value with username as a default value but give preference to env variable if set. does that sound ok?